### PR TITLE
Change logs debugging instructions

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: enterprise-registry/log-debugging/index.md [hunk 1]
```diff
@@ -11,29 +11,24 @@ weight: 5
...
 
-The aforementioned shell commands are also available in script form at https://github.com/coreos/docs/blob/master/enterprise-registry/log-debugging/tail-gunicorn-logs.sh
+Sign in to a super user account and visit `http://yourregister/superuser` to view the management panel:
+
+<img src="../build-support/superuser.png" class="img-center" alt="Enterprise Registry Management Panel"/>
+
+## View the logs for each service
+
+- Click the logs tab (<span class="fa fa-bug"></span>)
+- To view logs for each service, click the service name at the top. The logs will update automatically.
```
Comment (robszumski) [inline enterprise-registry/log-debugging/index.md]: Instead of "update", is "steam" a better verb in this context?

**Comment by robszumski** [comment]
Overall I tend to like title case for all of the headings, but I'll leave that up to your personal preference

File: enterprise-registry/log-debugging/index.md (hunk 1 above)
Comment (josephschorr) [inline enterprise-registry/log-debugging/index.md]: I prefer "sous vide" myself; better texture than steaming

(and yes, "stream" is probably better)

**Comment by robszumski** [comment]
Are there some common issues that we can list out here?

**Comment by josephschorr** [comment]
Not offhand. The setup tool's verification step seems to handle all the problems we've had reported in the past.

**Comment by robszumski** [comment]
Thats great to hear that we're catching errors before they become a problem. That leaves this doc kind of empty, do you think its worth keeping?

**Comment by josephschorr** [comment]
Yeah, so people still know where to go if they encounter a problem we haven't considered

**Pull request merged**